app/src/main/java/org/oppia/android/app/settings/profile/ProfileResetPinActivityPresenter.kt

```diff
@@ -31,7 +31,6 @@ class ProfileResetPinActivityPresenter @Inject constructor(
   }
 
   fun handleOnCreate() {
-    activity.title = activity.getString(R.string.profile_reset_pin_title)
```

Comment:
I checked the app again and it seems that we cannot remove this. This is the reason why your tests are failing. Add this back and tests should pass.